Makefile

```diff
@@ -18,6 +18,10 @@ build: ## Build site with production settings and put deliverables in ./public
 build-preview: ## Build site with drafts and future posts enabled
 	hugo --buildDrafts --buildFuture
 
+build-linkcheck: ## Build site with config for linkchecking.
+	rm -r public
+	hugo --minify --config config.toml,linkcheck-config.toml
```

Comment:
Q: Is there a reason to run `hugo` with `minify` flag?

Reply:
@kbhawkey no particular reason I wasn't sure why we use `minify` in production, and decided to be cautious and use it in this build.

Reply:
Makes sense to retain it, to me.

Reply:
That's fine. Just checking whether this flag is *required* or if the link checker works with HTML content generated when running `hugo server`.